conf/evolutions/default/13.sql

```diff
@@ -0,0 +1,7 @@
+# User
```

Comment:
💻 How can I apply database change? Should we schedule a maintenance??

Reply:
🧐 It’s a Jenkins job. The ALTER TABLE looks like an online DDL. I don’t think we need a maintenance.